test/e2e/scheduling/predicates.go

```diff
@@ -887,8 +887,8 @@ func initPausePod(f *framework.Framework, conf pausePodConfig) *v1.Pod {
 		ObjectMeta: metav1.ObjectMeta{
 			Name:            conf.Name,
 			Namespace:       conf.Namespace,
-			Labels:          conf.Labels,
-			Annotations:     conf.Annotations,
+			Labels:          map[string]string{},
```

Comment:
why is this change necessary?

Reply:
`testutils.StartPods` modifies Labels field. `Labels: conf.Labels` is a shallow copy and `conf.Labels` is a [shallow copy](https://github.com/kubernetes/kubernetes/blob/64c099d67069f85f2b9b78d7d12fd1a92fcc3d75/test/e2e/scheduling/priorities.go#L464) of `balancePodLabel`. Every call to `testutils.StartPods` then modifies `balancePodLabel` and disallows `balancePodLabel` to be used for listing all pods created for memory balancing.

Reply:
sg